recipe/meta.yaml

```diff
@@ -1,28 +1,30 @@
 {% set name = "click" %}
-{% set version = "8.1.8" %}
+{% set version = "8.2.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: ed53c9d8990d83c2a27deae68e4ee337473f6330c040a31d4225c9574d16096a
+  url: https://github.com/pallets/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: fd8f2d2cbdd3508a2b521abc9f4c72f25cf212057cfe8e2471d0ea7cfec14588
```

Comment:
We should keep PyPI as a source url because our automation tools also rely on it. We switch to github/gitlab only if additional files aren't available on PyPI.